businesscentral-monitoring/branch-overrides.yaml

```diff
@@ -14,20 +14,20 @@ modules:
     - name: jboss-eap-modules
       git:
         url: https://github.com/jboss-container-images/jboss-eap-modules.git
-        ref: EAP_7410_CR2
+        ref: EAP_7412_CR3
     - name: jboss-eap-7-image
       git:
         url: https://github.com/jboss-container-images/jboss-eap-7-image.git
-        ref: EAP_7410_CR2
+        ref: EAP_7412_CR3
     - name: wildfly-cekit-modules
       git:
         url: https://github.com/wildfly/wildfly-cekit-modules.git
         ref: 0.18.11
     - name: rhpam-7-image
       git:
         url: https://github.com/jboss-container-images/rhpam-7-image.git
-        ref: 7.13.x
```

Comment:
shouldn't this keep as 7.13.x as we are moving to 7.13.x branch?

Reply:
correct, updated.
thanks.